commit b3bece34956f86dcc8307f20b41a072ccdc917dc
Author: Christoph Hellwig <hch@lst.de>
Date:   Mon Apr 11 16:13:46 2022 +0200

    drm/i915/gvt: devirtualize ->inject_msi
    
    Just open code the MSI injection in a single place instead of going
    through the method table.
    
    Signed-off-by: Christoph Hellwig <hch@lst.de>
    Signed-off-by: Zhi Wang <zhi.a.wang@intel.com>
    Link: http://patchwork.freedesktop.org/patch/msgid/20220411141403.86980-18-hch@lst.de
    Reviewed-by: Jason Gunthorpe <jgg@nvidia.com>
    Reviewed-by: Zhi Wang <zhi.a.wang@intel.com>

diff --git a/drivers/gpu/drm/i915/gvt/hypercall.h b/drivers/gpu/drm/i915/gvt/hypercall.h
index 08c622c4079b..de63bd8dd05b 100644
--- a/drivers/gpu/drm/i915/gvt/hypercall.h
+++ b/drivers/gpu/drm/i915/gvt/hypercall.h
@@ -45,7 +45,6 @@ struct intel_vgpu;
 struct intel_gvt_mpt {
 	int (*host_init)(struct device *dev, void *gvt);
 	void (*host_exit)(struct device *dev, void *gvt);
-	int (*inject_msi)(struct intel_vgpu *vgpu, u32 addr, u16 data);
 	int (*enable_page_track)(struct intel_vgpu *vgpu, u64 gfn);
 	int (*disable_page_track)(struct intel_vgpu *vgpu, u64 gfn);
 	unsigned long (*gfn_to_mfn)(struct intel_vgpu *vgpu, unsigned long gfn);